src/manifest.json

```diff
@@ -3,17 +3,17 @@
   "description": "Replace the New Tab page with Bitcoin price chart",
   "version": "1.0.4",
   "icons": {
-      "16": "icons/icon16@3.png",
-      "48": "icons/icon48@3.png",
-      "128": "icons/icon128@3.png"
+    "16": "icons/icon16@3.png",
+    "48": "icons/icon48@3.png",
+    "128": "icons/icon128@3.png"
   },
   "chrome_url_overrides": {
     "newtab": "index.html"
   },
-  "manifest_version": 2,
-  "permissions": [
-    "https://bitcoin-price-api.devlabs-projects.info/**/*",
-    "https://api.crypto-tab.com/**/*",
-    "storage"
+  "manifest_version": 3,
+  "permissions": ["storage"],
+  "host_permissions": [
+    "https://bitcoin-price-api.devlabs-projects.info/",
+    "https://api.crypto-tab.com/"
```

Comment:
Changing `host_permissions` in a Chrome extension can cause the extension to be temporarily disabled for users when the extension is updated. This happens because Chrome treats changes to host permissions as significant, potentially security-impacting changes that require explicit user consent. When an extension requests new host permissions, Chrome disables the extension and prompts the user to accept the new permissions.

Can we keep the previous URL schemes in the new `host_permissions` (with the `/**/*` wildcard)?